### PR TITLE
Integrate sentry

### PR DESCRIPTION
**Pull request by Rup-Narayan-Rajbanshi**
## Changes
- Integrate sentry

### CODE REVIEW

File: risk_module/settings.py [hunk 1]
```diff
@@ -11,27 +11,51 @@
...
+    DATABASE_HOST=str,
+    TIME_ZONE=(str, 'UTC'),
+
+    USE_AWS_FOR_MEDIA=(bool, False),
+    S3_AWS_ACCESS_KEY_ID=str,
+    S3_AWS_SECRET_ACCESS_KEY=str,
+    S3_STORAGE_BUCKET_NAME=str,
+    S3_REGION_NAME=str,
+
+    CELERY_REDIS_URL=(str, 'redis://redis:6379/0'),
```
Comment (thenav56) [inline risk_module/settings.py]: Let's define this in docker-compose.yml instead
```suggestion
    CELERY_REDIS_URL=str,
```

File: risk_module/settings.py [hunk 2]
```diff
@@ -11,27 +11,51 @@
...
+
+    SENTRY_DSN=str,
```
Comment (thenav56) [inline risk_module/settings.py]:
```suggestion
    SENTRY_DSN=(str, None),
```

Comment (thenav56) [inline risk_module/settings.py]: This is a optional value
```suggestion
    SENTRY_DSN=(str, None),
```

File: risk_module/settings.py [hunk 3]
```diff
@@ -11,27 +11,51 @@
...
-# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+env = environ.Env(
+    DJANGO_DEBUG=(bool, False),
+    DJANGO_SECRET_KEY=str,
+    DJANGO_ALLOWED_HOST=(str, '*'),
+    # Database
+    DATABASE_NAME=str,
+    DATABASE_USER=str,
+    DATABASE_PASSWORD=str,
+    DATABASE_PORT=str,
```
Comment (thenav56) [inline risk_module/settings.py]:
```suggestion
    DATABASE_PORT=(int, 5432),
```

File: risk_module/settings.py [hunk 4]
```diff
@@ -11,27 +11,51 @@
...
-BASE_DIR = Path(__file__).resolve().parent.parent
+from risk_module import sentry
 
 
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+env = environ.Env(
+    DJANGO_DEBUG=(bool, False),
+    DJANGO_SECRET_KEY=str,
+    DJANGO_ALLOWED_HOST=(str, '*'),
```
Comment (thenav56) [inline risk_module/settings.py]:
```suggestion
    DJANGO_ALLOWED_HOSTS=(list, ['*']),
```

File: risk_module/settings.py [hunk 5]
```diff
@@ -11,27 +11,51 @@
...
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.environ.get('DJANGO_DEBUG', 'False').lower() == 'true'
+DEBUG = env('DJANGO_SECRET_KEY')
 
-ALLOWED_HOSTS = [os.environ.get('DJANGO_ALLOWED_HOST', 'localhost')]
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+ALLOWED_HOSTS = ['server', env('DJANGO_ALLOWED_HOST')]
```
Comment (thenav56) [inline risk_module/settings.py]:
```suggestion
ALLOWED_HOSTS = ['server', *env('DJANGO_ALLOWED_HOSTS')]
```

**Review by thenav56 — CHANGES_REQUESTED** [review]
Let's update docker-compose as well. For user-defined variables like DJANGO_SECRET_KEY let's not forget to use ?err

**Review by thenav56 — APPROVED** [review]

**Comment by thenav56** [comment]
@k9845 Please review and merge this.

**Review by k9845 — APPROVED** [review]

**Pull request merged**